classes/movie.class.php

```diff
@@ -18,7 +19,7 @@ public static function get_omdb_data($IMDBID, $Refresh = false) {
             if ($curl->error) {
             } else {
                 $OMDBData = json_encode($curl->response, JSON_UNESCAPED_UNICODE);
-                G::$DB->query("INSERT INTO movie_info_cache (IMDBID, OMDBData, OMDBTime) VALUES('$IMDBID', '" . db_string($OMDBData) . "', '" . sqlTime() . "')  ON DUPLICATE KEY UPDATE OMDBData='" . db_string($OMDBData) . "', OMDBTime='" . sqlTime() . "'");
+                G::$DB->query("INSERT INTO movie_info_cache (IMDBID, OMDBData, OMDBTime) VALUES('$IMDBID', '" . db_string($OMDBData) . "', '" . Time::sqlTime() . "')  ON DUPLICATE KEY UPDATE OMDBData='" . db_string($OMDBData) . "', OMDBTime='" . Time::sqlTime() . "'");
                 // query again? db_string escape?
```

Comment:
`sqlTime` is imported from the time class and is available in normal flow. But when the invocation comes from API, for some reason the method cannot be resolved.

Adding the class prefix seems to solve the issue and still works with the original flow.

Is there any init similar to 
> should add ImageTools::init() in api/index.php to avoid null provider.
that I'm missing?